plugins/auth/token.js

```diff
@@ -16,7 +16,7 @@ module.exports = () => ({
         notes: 'Generate a JWT for use throughout Screwdriver',
         tags: ['api', 'auth', 'token'],
         auth: {
-            strategies: ['token', 'session', 'auth_token'],
+            strategies: ['session', 'auth_token'],
```

Comment:
Potentially should be in its own PR? Not really really related to the rest of this one.